extramodules/it/eng/jhove/module/png/PngModule.java

```diff
@@ -961,7 +960,7 @@ private final void checkIHDR(final DataInputStream inputStream,
 													colorType + ": " +tmp ));
 
 			}
-			repInfo.setProfile("PNG GrayScale");
+			repInfo.setProfile(PNG_PROFILES[0]);
```

Comment:
From here onwards are all improvements regardless as the strings are so long. I suspect that there's a better way of setting these up in the first place using enums but that's beyond the scope of this work and this is the first step.

Reply:
Thanks, Carl. Java is still new to me so if you think of a better way to address these categorized strings, let me know. Going forward should I continue to set up constants this way (for instance, there are some repeated error messages in this file that I am thinking of using the enums Error_Message[] for, but  should just use a separate constant for each error message?)